app/controllers/tasuku/taskables/questions/answers_controller.rb

```diff
@@ -6,6 +6,30 @@ class Taskables::Questions::AnswersController < ApplicationController
 
     before_action :set_question
 
+    def update
+      answer = Taskables::Question::Answer.find(params[:id])
+
+      return redirect_to :back, alert: I18n.t('tasuku.taskables.questions.answers.no_answers') unless params[:taskables_question_answer]
+
+      # Delete the old votes
```

Comment:
Completely agree, originally this was way more complicated. However i opted for just deleting all the old answers instead of keeping the ones that where resubmitted.